documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -31,19 +51,33 @@ kind: Secret
 metadata:
   name: <secret>
   namespace: {namespace}
+  ownerReferences:
+    - apiVersion: forklift.konveyor.io/v1beta1
+      kind: Provider
+      name: <provider_name>
+  labels:
+    createdForProviderType: <provider_type> <1>
 type: Opaque
 stringData:
-  user: <user> <1>
-  password: <password> <2>
-  cacert: | <3>
-    <engine_ca_certificate>
-  thumbprint: <vcenter_fingerprint> <4>
-EOF
+  user: <user> <2>
+  password: <password> <3>
+  domainName: <domain_name> <4>
+  projectName: <project_name> <5>
+  regionName: <region name> <6>
+  cacert: | <7>
+    <ca_certificate>
+  url: <api_end_point> <8>
+  thumbprint: <vcenter_fingerprint> <9>
 ----
-<1> Specify the vCenter user or the {rhv-short} {manager} user.
-<2> Specify the user password.
-<3> {rhv-short} only: Specify the CA certificate of the {manager}. You can retrieve it at `https://<engine_host>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA`.
-<4> VMware only: Specify the vCenter SHA-1 fingerprint.
+<1>  Specify the type of source provider. Allowed values are `ovirt`, `vsphere`, and `openstack`. This label is needed to verify the credentials are correct when the remote system is accessible and, for {rhv-short}, to retrieve the {manager} CA certificate when a third-party certificate is specified.
+<2> Specify the vCenter user, the {rhv-short} {manager} user, or the {osp} user.
+<3> Specify the user password.
+<4> {osp} only: Specify the domain name.
+<5> {osp} only: Specify the project name.
+<6> {osp} only: Specify the name of the {osp} region.
+<7> {rhv-short} only: {manager} CA certificate unless it was replaced by a third-party certificate, in which case enter the {manager} Apache CA certificate. You can retrieve the {manager} CA certificate at https://<engine_host>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA.
```

Comment:
OpenStack provider also has this cacert option.

Reply:
Fixed.